includes/footer.php

```diff
@@ -77,6 +77,9 @@
             <a href="https://twitter.com/nf_core" target="_blank" title="Twitter" data-bs-toggle="tooltip">
               <img src="/assets/img/twitter.svg" />
             </a>
+            <a href="https://mstdn.science/@nf_core" target="_blank" title="mastodon" data-bs-toggle="tooltip">
```

Comment:
This won't work, as you missed out the `rel="me"` which Mastodon needs.

My PR is clearly superior 😉  😆 https://github.com/nf-core/nf-co.re/pull/1448